registry.go

```diff
@@ -89,8 +89,7 @@ func main() {
 	if err := cmd.Start(); err != nil {
 		log.Fatalf("Could not start node: %s", err)
 	}
-	// TODO: Does defer even work here?
-	defer func() {
+	go func() {
 		if err := cmd.Wait(); err != nil {
```

Comment:
If the node process exits gracefully, `err == nil`, so we won't log it.  It seems like if the node process exits *at*all* (gracefully or not), that we should die so both node and the Go daemon are restarted.